addons/hw_drivers/iot_handlers/drivers/PrinterDriver_L.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, identifier, device):
 
         if any(cmd in device['device-id'] for cmd in ['CMD:STAR;', 'CMD:ESC/POS;']):
             self.device_subtype = "receipt_printer"
-        elif "COMMAND SET:ZPL;" in device['device-id']:
+        elif "COMMAND SET:ZPL;" in device['device-id'] or "CMD:ESCLABEL;" in device['device-id']:
```

Comment:
```suggestion
        elif device['device-id'] in ("COMMAND SET:ZPL;",  "CMD:ESCLABEL;"):
```

Reply:
better to have this syntax if it is likely than in the future we add up other printers id